Justfile

```diff
@@ -39,6 +39,7 @@ bind-kotlin: setup
     generate --library bound/kt/src/main/resources/libtbdex_uniffi_aarch64_apple_darwin.dylib \
     --language kotlin \
     --out-dir target/bindgen-kotlin
+  sed -i '' 's/findLibraryName(componentName)/detectSystemTarget()/' target/bindgen-kotlin/tbdex/sdk/rust/tbdex.kt
```

Comment:
Do you need to reflect this in the local build commands as well or nah?

Reply:
nope, just here

I have an action item to consider improvement around the `just bind` [here](https://github.com/TBD54566975/tbdex-rs/issues/108#issuecomment-2273491010)